### PR TITLE
message fixes

### PR DESCRIPTION
**Pull request by shriyanka**

### CODE REVIEW

File: HTTP_200/settings.py [hunk 1]
```diff
@@ -32,7 +32,7 @@
 # Application definition
 
 INSTALLED_APPS = (
-    # 'rest_framework',
+    'rest_framework',
```
Comment (DESHRAJ) [inline HTTP_200/settings.py]: @shriyanka any reason for uncommenting this ?? I think we dont need it as of now.

**Comment by DESHRAJ** [comment]
@shriyanka please remove the conflicts and then we are good to go.

**Comment by shriyanka** [comment]
@deshraj was using that in message framework. 
rest is so good that i unknowingly created the api's in that

**Comment by shriyanka** [comment]
@DESHRAJ  done you can merge this up :+1:

**Comment by shriyanka** [comment]
Closing it. will ReOpen after fixing names

**Pull request closed without merging**